npm/ux/scatterChart.tsx

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  CreateIntegerTicks,
+  CreateTimeTicks,
+  Tick,
+} from "qsharp-lang/ux/ticks";
```

Comment:
I think this should just be a relative path to the source, then you can remove the package.json changes to exports entirely, i.e.

`} from "../src/ux/ticks.js"`

(Note: Still use a .js extension and it will resolve).

This way, the unit tests in basics.js are still testing the compiled source under 'dist', but any package importing the components as source are still only pulling source, and aren't pulling a mix of source tsx and built js for the ux components.

Not sure if that makes sense, but try it. (Or leave as-is, and I'll do a follow up)

Reply:
Let me try

Reply:
Seems to work. Thank you, @billti !